src/panels/config/entities/entity-registry-basic-editor.ts

```diff
@@ -123,16 +122,16 @@ export class HaEntityRegistryBasicEditor extends SubscribeMixin(LitElement) {
       computeDomain(this.entry.entity_id);
 
     return html`
-      <paper-input
+      <ha-textfield
+        error-message="Domain needs to stay the same"
```

Comment:
This is not a property for textfield https://github.com/material-components/material-web/tree/master/packages/textfield#propertiesattributes